packages/kbn-build/src/commands/bootstrap.test.js

```diff
@@ -0,0 +1,163 @@
+jest.mock('../utils/scripts', () => ({
+  installInDir: jest.fn(),
+  runScriptInPackageStreaming: jest.fn(),
+}));
+
+import path from 'path';
+
+import { run } from './bootstrap';
+import { Project } from '../utils/project';
+import { buildProjectGraph } from '../utils/projects';
+import { installInDir, runScriptInPackageStreaming } from '../utils/scripts';
+
+const createProject = (fields, path = '.') =>
+  new Project(
+    {
+      name: 'kibana',
+      version: '1.0.0',
+      ...fields,
+    },
+    path
+  );
+
+beforeEach(() => {
+  installInDir.mockReset();
```

Comment:
`jest.resetAllMocks()`? I'm surprised to find that we're not using https://facebook.github.io/jest/docs/en/configuration.html#resetmocks-boolean

Reply:
There should be absolutely no reason to persist mock state between tests

Reply:
++ definitely agreed (I just forget), fixing